### PR TITLE
engagement recency

### PR DESCRIPTION
**Pull request by avigoldman**
### Support Docs Update Checklist

Note: when you open a pull request, your changes are published to https://staging.sparkpost.com/docs.

Please take a moment to check each of these steps for your pull request:

- [ ] Give your pull request a meaningful name.
- [ ] Include a description of your changes in the pull request.
- [ ] _SparkPost staff only_: Check that your article appears [here](https://staging.sparkpost.com/docs) and looks correct.
- [ ] Check the links in your article.
- [ ] Use lowercase filenames.

### CODE REVIEW

File: articles/reporting/engagement-recency.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+![Engagement Recency details page](media/engagement-recency/details.png)
+
+With this report, you can see if you are sending to too many disengaged recipients. If the Never Engaged and Not Recently Engaged cohorts make up a majority of the people you send to, you should examine your [list hygiene practices](https://www.sparkpost.com/docs/signals/list-hygiene/).
+
+### Behavior by Cohort
+
+Knowing which cohorts your sending to is an important step in understanding the health of your email program. You can further dive into each group of recipients to optimize your sending and messaging, as well as find areas for improvement. The three Behavior by Engagement Cohorts reports provide metrics about how each of your different cohorts are engaging with your email, as well as suggestions on how to interpret them. For example, if you see a high unsubscribe or complaint rate from the New cohort, you probably need to work on your list collection and set better expections with new recipients when you collect their emails.
+
+48 hours after emails are sent, engagement is processed to give you an accurate view into their patterns.
```
Comment (jgzamora) [inline articles/reporting/engagement-recency.md]: "Recipient engagement is processed 48 hours after emails are sent to give you an accurate view into their patterns."

I think this sentence belongs somewhere above in the doc. It applies to all charts no?

File: articles/reporting/engagement-recency.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+In the example below, you can see 51.2% of the email that were sent on August 16th were sent to recipients who engaged with this sender's emails in the past 14 days.
+
+![Engagement Recency details page](media/engagement-recency/details.png)
+
+With this report, you can see if you are sending to too many disengaged recipients. If the Never Engaged and Not Recently Engaged cohorts make up a majority of the people you send to, you should examine your [list hygiene practices](https://www.sparkpost.com/docs/signals/list-hygiene/).
+
+### Behavior by Cohort
+
+Knowing which cohorts your sending to is an important step in understanding the health of your email program. You can further dive into each group of recipients to optimize your sending and messaging, as well as find areas for improvement. The three Behavior by Engagement Cohorts reports provide metrics about how each of your different cohorts are engaging with your email, as well as suggestions on how to interpret them. For example, if you see a high unsubscribe or complaint rate from the New cohort, you probably need to work on your list collection and set better expections with new recipients when you collect their emails.
```
Comment (jgzamora) [inline articles/reporting/engagement-recency.md]: - I'd lose the first sentence / Behavior by Engagement Cohorts
 
You can further dive into each group of recipients to optimize your sending and find areas for improvement. The next three reports show how your different cohorts are engaging with your emails, as well as recommendations on how to interpret their behavior. For example, if you see a high unsubscribe or complaint rate from the New cohort, you probably need to work on your list collection and set better expectations with new recipients when you collect their emails.

Comment (jgzamora) [reply articles/reporting/engagement-recency.md]: I think also maybe the recommendations could either be omitted or have their own section.

File: articles/reporting/engagement-recency.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+48 hours after emails are sent, engagement is processed to give you an accurate view into their patterns.
+
+The behavior chars are:
```
Comment (jgzamora) [inline articles/reporting/engagement-recency.md]: charts?

File: articles/reporting/engagement-recency.md (hunk 1 above)
Comment (avigoldman) [reply articles/reporting/engagement-recency.md]: nope, just the behavior charts

File: articles/reporting/engagement-recency.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+![Engagement Recency dashboard](media/engagement-recency/dashboard.png)
+
+## Details page
+
+On each details page you'll find 4 different charts, each in a different tab.
+
+The first chart shows the total emails you sent each day, broken up by each cohort.
+
+In the example below, you can see 51.2% of the email that were sent on August 16th were sent to recipients who engaged with this sender's emails in the past 14 days.
```
[the remark this answers is not in the document]
Comment (avigoldman) [reply articles/reporting/engagement-recency.md]: this is a good call - that way the reader has seen the example before reading the description

**Pull request merged**